pkg/e2e/framework/common.go

```diff
@@ -273,6 +273,61 @@ type MachineSetParams struct {
 	ProviderSpec *mapiv1beta1.ProviderSpec
 }
 
+// CreateMachine creates a new Machine resource.
+func CreateMachine(c client.Client, params MachineSetParams) (*mapiv1beta1.Machine, error) {
+	if params.Labels == nil {
+		params.Labels = make(map[string]string)
+	}
+
+	// TODO(bison): It would be nice to automatically set the Cluster ID / name
+	// in the labels here, but I'm not sure how to easily find it.
+	params.Labels["e2e.openshift.io"] =
+		fmt.Sprintf("%s-%d", params.Name, time.Now().Unix())
+
+	machine := &mapiv1beta1.Machine{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Machine",
+			APIVersion: "machine.openshift.io/v1beta1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      params.Name,
+			Namespace: MachineAPINamespace,
+			Labels:    params.Labels,
+		},
+		Spec: mapiv1beta1.MachineSpec{
+			ProviderSpec: *params.ProviderSpec,
+			ObjectMeta: metav1.ObjectMeta{
+				Labels: params.Labels,
+			},
+		},
+	}
+
+	if err := c.Create(context.Background(), machine); err != nil {
+		return nil, err
+	}
+
+	return machine, nil
+}
+
+// WaitForMachine waits for the machine to enter the "Running" phase and for node to be ready
+func WaitForMachine(c client.Client, name string) {
+	Eventually(func() error {
+		machine, err := GetMachine(c, name)
+		Expect(err).ToNot(HaveOccurred())
+
```

Comment:
I think we should to validate the machine has no deletion timestamp here, and also probably check phase is running.
But base on https://github.com/openshift/cluster-api-actuator-pkg/pull/128/files#r363689909 may be we don't need this two methods